tests/interpreter/cases/call/or.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+cases:
+  - note: or-error
+    data: {}
+    modules:
+      - |
+        package test
+
+        inc(x) = x + 1
+        inc(x) = x + 4
+
+        a1 = inc(5)
+    query: data.test
+    error: functions must not produce multiple outputs for same inputs
+
+  - note: or-ok
+    data: {}
+    modules:
+      - |
+        package test
+
+        inc(x) = x + 1
+        inc(x) = x + 1
+
+        a1 = inc(5)
+    query: data.test
+    want_result:
+      a1: 6
+
+  - note: or-true
+    data: {}
+    modules:
+      - |
+        package test
+
+        inc(x) = 1 == x
+        inc(x) = 2 == x
+
+        a1 = inc(2)
+    query: data.test
+    want_result:
+      a1: true
```

Comment:
The above example produces `eval_conflict_error: functions must not produce multiple outputs for same inputs` in Rego playground.